### ISSUE TITLE
Migrate existing infra to Terraform 

### ISSUE THREAD

**Issue by pilgrim2go** [hashicorp/terraform#3730]
Hi,

Is there a way to migrate existing infra to Terraform?

I tried Terraform. Love it and plan to migrate to Terraform.

Thanks,
Van

**Comment by rmenn** [comment]
Have you tried terraframe? It does the job.

**Comment by rmenn** [comment]
Sorry i think this is what you need to check out. https://github.com/dtan4/terraforming

**Comment by radeksimko** [comment]
Here's the original issue discussing the feature: https://github.com/hashicorp/terraform/issues/581 which is why I'm closing this issue, so we have one place where we discuss this.

Also there are some PRs you may want to watch:
https://github.com/hashicorp/terraform/pull/2022
https://github.com/hashicorp/terraform/pull/3345

TL;DR It is _currently_ not possible to migrate existing infra with terraform itself (the project @rmenn mentioned may help you though), but it's one of the most wanted features.
